plugins/node-team/scripts/lib/team.sh

```
@@ -0,0 +1,295 @@
+      TEAM_NAME="Node Core"
+      TEAM_SPRINT_FILTER="Node Core"
+      if [[ -f "${na_dir}/team-roster-core.json" ]]; then
+        TEAM_ROSTER_FILE="${na_dir}/team-roster-core.json"
+      else
+        TEAM_ROSTER_FILE="${root_dir}/config/team-roster-core.json"
+      fi
+      TEAM_BUG_COMPONENTS='"Node", "Node / CRI-O", "Node / Kubelet", "Node / CPU manager", "Node / Memory manager", "Node / Topology manager", "Node / Numa aware Scheduling", "Node / Device Manager", "Node / Pod resource API", "Node / Node Problem Detector", "Node / Kueue", "Node / Instaslice-operator"'
+      TEAM_BACKLOG_KEYWORDS=""
+      ;;
+    "all"|"All"|"All Node")
+      TEAM_NAME="All Node"
```

Shellcheck (0.11.0)

[warning] TEAM_NAME appears unused. Verify use (or export if used externally).

(SC2034)